ci-operator/step-registry/gcp/provision/buildhost/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
How about creating a plain text file for the OWNERS file instead of symbolic link file, that will avoid affecting other unrelated steps OWNERS change

Reply:
@wgahnagl This is not resolved yet, and you need to list you as the OWNER of the step.